src/test/cli/rbd/not-enough-args.t

```diff
@@ -188,6 +188,7 @@
   rbd: locker was not specified
   [22]
   $ rbd bench-write
+  rbd: bench-write is deprecated, use rbd bench --io-type write ...
```

Comment:
wait, I am sure I have fixed this one, and it passed the run-cli-tests. Let me check it again.